PurchasesTests/Purchasing/EntitlementInfosTests.swift

```diff
@@ -441,6 +441,30 @@ class EntitlementInfosTests: XCTestCase {
         verifyProduct()
     }
 
+    func testSubscriptionIsSandboxString() {
```

Comment:
👍

Reply:
- removed since we now are sure we're getting a `bool` and not a string for this value.